core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -247,6 +250,24 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
       .createAzureStorageContainer(requestBody, workspaceId)
   }
 
+  override def createGcpStorageBucket(workspaceId: UUID,
+                                      storageBucketName: String,
+                                      location: String,
+                                      ctx: RawlsRequestContext
+  ): CreatedControlledGcpGcsBucket = {
+    val creationParams =
+      new GcpGcsBucketCreationParameters().name(storageBucketName).location(location)
```

Comment:
Will WSM buckets need to support any additional features that we use today, like autoclass or CORS?

Reply:
Yes, see this [code](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala#L192). I would prefer to factor that into WSM if possible rather than have Rawls do that out of band.